### PR TITLE
Improve README for the planned model usage

### PR DESCRIPTION
**Pull request by jamietanna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 Repository containing all work related to the [Data Standards Authority's](https://alphagov.github.io/data-standards-authority/) work towards producing a federated API discovery model for UK Government.
 
+This model aims to allow for inter- and intra-agency discovery of APIs that are available, separate to their API documentation or specifications, as a means to understand what APIs are available.
```
Comment (charlesbaird-dsa) [inline README.md:5]:
```suggestion
This model aims to allow for inter- and intra-agency discovery of APIs, separate to their API documentation or specifications, as a means to understand what APIs are available and to make it easier for users to find them.
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
...
+
+This model is planned to be push- and pull-based, allowing the use of different catalogues across government, as well as the [public API Catalogue on www.api.gov.uk](https://www.api.gov.uk).
+
+The aim of this model is that it promotes privacy of APIs, not defaulting to exposing information about every API to the public, or to other catalogues that are connected, without being opted in by the owner of the API.
```
Comment (charlesbaird-dsa) [inline README.md:9]:
```suggestion
The aim of this model is that it promotes standardisation of data shared about APIs - to reduce privacy and security concerns by sharing a minimal data set, and therefore increase the number of APIs which organisations feel comfortable publishing details about.
```

**Comment by jamietanna** [comment]
Thanks for the tweaks @astreater @charlesbaird-dsa - mind re-reviewing?

**Review by charlesbaird-dsa — APPROVED** [review]
LGTM

**Pull request merged**